src/server/accessors/Notifier.ts

```diff
@@ -46,4 +71,27 @@ export class Notifier implements INotifier {
     public getMessageBuilder(): IMessageBuilder {
         return new MessageBuilder();
     }
+
+    /**
+     * Creates a new direct message room between two users.
+     *
+     * @param {IUser} user - The first user to create the room for.
+     * @param {IUser} sender - The second user to create the room for.
+     * @returns {Promise<IRoom>} A Promise that resolves with the newly created direct message room.
+     */
+    private async createDMRoom(user: IUser, sender: IUser): Promise<IRoom> {
+        const newDMRId = await this.roomBridge.doCreate(
+            {
+                type: RoomType.DIRECT_MESSAGE,
+                usernames: [user.username, sender.username],
```

Comment:
dont think is required/used

Reply:
It is required, or TS errors pop up. I think this is planned to be removed in `2.0.0`

Reply:
Same as the last comment .. but maybe also pass an empty array instead as it's not used anyway.